.github/workflows/testflight.yml

```diff
@@ -0,0 +1,64 @@
+name: Upload to Testflight
+on:
+  push:
+    branches:
+      - master
+      - ci/** # Todo remove before merging
+
+env:
+  FASTLANE_KEYCHAIN_NAME: fastlane.keychain
+
+jobs:
+
+  upload_to_testflight:
+    name: Build and Upload iOS-Swift to Testflight
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./scripts/ci-select-xcode.sh
+
+      # The Github Actions hang when Xcode is trying to sign the app because
+      # a modal window pops up to access the signing key's keychain. This modal
+      # lets the action hang. A workaround is to create a local keychain and
+      # store the signing keys in there.
+      # https://github.com/actions/virtual-environments/issues/1820#issuecomment-719549887
+      
+      # Sometimes the keychain already exists
+      - name: Delete local keychain
+        continue-on-error: true
+        run: security delete-keychain $FASTLANE_KEYCHAIN_NAME
+          
+      - name: Create local keychain
+        run: |
+          security create-keychain -p ${{ secrets.FASTLANE_KEYCHAIN_PASSWORD }} $FASTLANE_KEYCHAIN_NAME
```

Comment:
Are these passwords only for the specific test app? What can one do with these secrets?

Reply:
Maybe we should rather discuss in private what you can do with these secrets.

Reply:
Maybe lets have a _secret_ meeting 🤫

Reply:
let's do it over signal :D